### PR TITLE
Combo

### PR DESCRIPTION
**Pull request by Albinka-811**

### CODE REVIEW

File: src/styles/components/combo-set.scss [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+  display: flex;
+  overflow: hidden;
+  height: 562px;
+  margin: 64px 0 0 130px;
+
+  &-next {
+    cursor: pointer;
+    position: absolute;
+    top: 50%;
+    margin-top: -40px;
```
Comment (Anatolych97) [inline src/styles/components/combo-set.scss]: Use transformations

File: src/styles/components/combo-set.scss [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    transition: 0.6s ease;
+    user-select: none;
+
+    &_arrow {
```
Comment (Anatolych97) [inline src/styles/components/combo-set.scss]: Wrong selector name

**Review by Anatolych97 — CHANGES_REQUESTED** [review]

File: src/styles/components/combo-set.scss (hunk 1 above)
Comment (Albinka-811) [reply src/styles/components/combo-set.scss]: fixed

File: src/styles/components/combo-set.scss (hunk 2 above)
Comment (Albinka-811) [reply src/styles/components/combo-set.scss]: correct

**Review by Anatolych97 — APPROVED** [review]

**Pull request merged**